web-api-db-ctrl/src/postgres.rs

```diff
@@ -36,19 +36,19 @@ use crate::Error;
 static MT_USER: Lazy<QuotedIdentifier> = Lazy::new(|| "web-api-mt".parse().unwrap());
 
 static PUBLIC_SCHEMA_MIGRATOR: Lazy<Migrator> = Lazy::new(|| {
-    let mut migrator = sqlx::migrate!("migrations/public");
+    let mut migrator = sqlx::migrate!("postgres/public");
```

Comment:
I changed the folder name for consistency with elastic search. 

Now schemas for elastic-search are under `elastic-search/` and schemas for postgres are under `postgres/`